test/fixtures/toc.json

```diff
@@ -1355,18 +1355,6 @@
       "cli-adding-kvm-details-to-a-compute-profile_kvm-provisioning",
       "cli-creating-network-or-image-based-hosts_kvm-provisioning"
     ],
-    "provisioning_virtual_machines_on_ovirt_rhv-provisioning": [
```

Comment:
You don't need to change this file manually. It will be changed automatically when the documentation team will remove those topics from the documentation.